applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/unicast_client.c

```diff
@@ -665,16 +753,19 @@ static bool source_parse_cb(struct bt_data *data, void *user_data)
  * @param[in]	dir		Direction of the capabilities to check.
  * @param[in]	index		Device index.
  *
- * @return	True if valid codec capability found, false otherwise.
+ * @retval	0		No valid codec capabilities found.
+ * @retval	Other	Bitfield of valid codec capabilities.
  */
-static bool valid_codec_cap_check(struct bt_audio_codec_cap cap_array[], uint8_t num_caps,
-				  enum bt_audio_dir dir, uint8_t index)
+static uint32_t valid_codec_cap_check(struct bt_audio_codec_cap cap_array[], uint8_t num_caps,
```

Comment:
We should rename to 
```suggestion
static uint32_t valid_codec_capab_check(
```
It is confusing wrt. LE Audio CAP.